cloud_edition/flexibility_mode/docs/crontasks.rst

```diff
@@ -0,0 +1,56 @@
+Crontasks settings
+==================
+
+You find here, some good practices about crontasks management in Akeneo PIM Flexibility Edition :
+
+1. CRONs we provide by default uses a custom wrapper script defined by *SHELL* that call the PIM "bin/console" command.
+
+   It avoid us to write on each cron line the full script path to call and the redirection of logs.
+
+   Adding your own cron lines failes because it tries to run your command with our wrapper.
```

Comment:
fail

Reply:
avoids
fails

Reply:
👌

Reply:
`However, be careful, when adding your own cron line. You could if you are calling a PIM's Symfony command.`